HACKING.jst.md

```diff
@@ -186,6 +162,22 @@ is likely that ocamlformat has crashed (e.g. with a parser error) while looking
 at your test. The cryptic error output will mention the name of the test. Run
 ocamlformat manually on your test file to see the actual error.
 
+Debugging
+---------
+
+- To see the `parser-extended` ast coming out of the parser for a given source
+  file, use `dune exec -- tools/printast/printast.exe path/to/your/file.ml`.
```

Comment:
Are both of these `dune exec` lines correct? One has the executable after the `--` and the other has it before. If you double-check this and believe it's correct, I'll believe it.

Reply:
Well spotted.  It turns out both do work!  But I've changed the one I copied from elsewhere in the doc to be more idiomatic.